build.gradle

```diff
@@ -6,12 +6,14 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath 'com.android.tools.build:gradle:2.3.0'
+        classpath 'com.google.gms:google-services:3.1.1'
     }
 }
 
 allprojects {
     repositories {
         jcenter()
+        maven { url "https://maven.google.com" }
```

Comment:
why maven?

Reply:
It's requested by FCM compile: https://github.com/firebase/quickstart-android/blob/a9227f0b8cbf941402bc6b3fd90ab449e8be9e56/messaging/build.gradle#L23

Reply:
ok